### ISSUE TITLE
OSC indicates cached data with incorrect order of magnitude

### ISSUE THREAD

**Issue by foxdevilwild** [TheAMM/mpv_thumbnail_script#30]
The OSC's cache display appears to be off by an order of magnitude - probably due to assuming the cache size is set in bytes as opposed to KiB.

Steps to reproduce:
Configure mpv's cache with a size of n MiB and replay a sufficiently large video file.

Expected behaviour:
mpv_thumbnail_script_client_osc should indicate a cache size of roughly n MiB

Current behaviour:
mpv_thumbnail_script_client_osc indicates a cache size of roughly n KiB (as opposed to MiB)

versions: mpv 0.27.2 / mpv_thumbnail_script 0.4.2

I have confirmed that, using the same configuration, mpv's default OSC shows the correct cache size. Does anybody else encounter this issue? If this is not due to my (slightly) out of date version of mpv, I'd be happy to provide a PR which fixes the issue.

Thank you for this overall fantastic extension.

**Comment by TheAMM** [comment]
I can't confirm, as the original osc.lua [dropped the cache size in January](https://github.com/mpv-player/mpv/commit/3a76bfdccc83e928fe9e885b32e8dda3a10c69ae#diff-5d10e79e2d65d30d34f98349f4ed08e4) (although it was too late for [0.27.2](https://github.com/mpv-player/mpv/commits/v0.27.2/player/lua/osc.lua)).
I don't know what arguments you were using, but [patched_osc.lua seems to be doing the right thing](https://github.com/TheAMM/mpv_thumbnail_script/blob/master/src/patched_osc.lua#L2089) ([`cache-used` is KiB](https://mpv.io/manual/master/#command-interface-cache-used)).
The patched OSC is supposed to be a direct copy of the original, but with a few added hooks to render the thumbnails, so I'm not too eager to add specific fixes for old mpv versions. I haven't updated it in a year, but [there hasn't been much changed either](https://github.com/mpv-player/mpv/commits/master/player/lua/osc.lua), other than dropping the cache size display.

In short, the issue can be considered outdated and if there was an error, it was likely from the original file (or from version mismatch etc). You can edit your copy of the script to display the cache size properly, although everything *should* be correct as-is.
